.github/workflows/ci-e2e.yaml

```diff
@@ -58,14 +65,29 @@ jobs:
           make kind-load-controller deploy-controller
           kubectl --context kind-mgc-control-plane -n multicluster-gateway-controller-system wait --timeout=300s --for=condition=Available deployment/mgc-controller-manager
           kubectl --context kind-mgc-control-plane logs --all-containers --ignore-errors deployment/mgc-controller-manager -n multicluster-gateway-controller-system
-      - name: Run suite
+          kubectl get managedzones -n multi-cluster-gateways mgc-dev-mz-aws -o yaml
+          kubectl --context kind-mgc-control-plane -n multi-cluster-gateways wait --timeout=60s --for=condition=Ready managedzone/mgc-dev-mz-aws
+          kubectl get managedzones -n multi-cluster-gateways mgc-dev-mz-gcp -o yaml
+          kubectl --context kind-mgc-control-plane -n multi-cluster-gateways wait --timeout=60s --for=condition=Ready managedzone/mgc-dev-mz-gcp
+      - name: Run suite AWS
         run: |
           export OCM_SINGLE=1
           export TEST_HUB_NAMESPACE=multi-cluster-gateways
           export TEST_HUB_KUBE_CONTEXT=kind-mgc-control-plane
           export TEST_MANAGED_ZONE=${{ secrets.E2E_AWS_DNS_PUBLIC_ZONE_NAME }}
           make test-e2e
-      - name: Dump MGC logs
+      - name: Dump MGC logs AWS
+        if: ${{ failure() }}
+        run: |
+          kubectl --context kind-mgc-control-plane logs --all-containers --ignore-errors deployment/mgc-controller-manager -n multicluster-gateway-controller-system    
```

Comment:
You can probably drop the extra `Dump MGC logs x` and just have it once as the last step. It only needs to run once on failure and it will skip all other steps anyway.